File: xyz/Timus/1001/src/Solution.cpp
#include <cmath>
#include <iomanip>
#include <iostream>
#include <stack>

using namespace std;

int main() {
    stack<long long> s;

    long long n;

    while (cin >> n) {
        s.push(n);
    }

    while (!s.empty()) {
        cout << fixed << setprecision(4) << sqrt(static_cast<double>(s.top())) << endl;
        s.pop();
    }
}

File: xyz/Timus/1001/src/Solution.java
import java.util.Scanner;
import java.util.Stack;

public class Solution {

    public static void main(String[] args) {
        try (Scanner input = new Scanner(System.in, "UTF-8")) {
            Stack<Long> s = new Stack<>();

            long n;

            while (input.hasNext()) {
                n = input.nextLong();
                s.push(n);
            }

            while (!s.empty()) {
                System.out.printf("%.4f%n", Math.sqrt((double) s.peek()));
                s.pop();
            }
        }
    }
}
